src/FormBuilder/resources/js/form-builder/src/settings/group-email-settings/email/logo-upload/index.tsx

```diff
@@ -41,20 +41,36 @@ export default ({value, onChange}) => {
         frame.open();
     };
     return (
-        <div style={{display: 'flex', flexDirection: 'column', marginBottom: '8px', width: '100%'}}>
-            <div>
-                {' '}
-                {/* Wrapping the TextControl solves a spacing issue */}
-                <TextControl type={'url'} label={__('Logo URL', 'give')} value={value} onChange={onChange} />
+        <BaseControl
+            label={__('Logo URL', 'give')}
+            help={__(
+                'Upload or choose a logo to be displayed at the top of the donation receipt emails. Displayed on HTML emails only.',
+                'give'
+            )}
+        >
+            <div style={{display: 'flex', gap: '0.25rem'}}>
+                <input
+                    type={'url'}
+                    value={value}
+                    onChange={onChange}
+                    className={'components-text-control__input'}
+                    style={{flex: 1}}
+                />
+                <Button
+                    icon={upload}
+                    variant={'secondary'}
+                    onClick={openMediaLibrary}
+                    style={{
+                        borderRadius: '0.125rem',
+                        paddingLeft: '1rem',
+                        paddingRight: '1rem',
+                        flex: '0 0 auto',
+                        height: 'auto',
+                    }}
```

Comment:
Please move the styles into css instead of inline. Same with above.

Reply:
Resolved: d3cee64fe6829f0e7a07fba58c1ce07da0230f13